test/framework/fileConversion.go

```diff
@@ -112,17 +155,31 @@ func doCmd(osCmd string, osArgs ...string) error {
 	return nil
 }
 
-// copyIfNotPresent checks for the src file in the tgtDir.  If it is not there, it attempts to copy if from src to tgtdir.
+// copyIfNotPresent checks for the src file in the tgtDir.  If it is not there, it attempts to copy it from src to tgtdir.
 // If a copy is performed, the path to the copy is returned.
 // If no copy is performed, the original src string is returned.
 func copyIfNotPresent(src, tgtDir string) (string, error) {
-	base := filepath.Base(src)
-	// Only copy the source image if it does not exist in the temp directory
-	if _, err := os.Stat(filepath.Join(tgtDir, base)); err != nil {
-		if err := doCmd("cp", "-f", src, tgtDir); err != nil {
+	ret := filepath.Join(tgtDir, filepath.Base(src))
```

Comment:
This func does not exactly implement it's comment (last comment). Me, personally, I'd refactor as:
```
// If no copy is performed, the original src string is returned.  <========= true??
func copyIfNotPresent(src, tgtDir string) (string, error) {
	ret := filepath.Join(tgtDir, filepath.Base(src))
	err := os.Stat(ret)
	if  err == nil { // file exists, don't copy
                return src, nil  // do we return src per comment or should ret be returned??
        }
	if !os.IsNotExist(err) {
		return src, errors.Wrap(err, "Unexpected error stating file")
	}
	if err = doCmd("cp", src, ret); err != nil {
			return src, err
	}
	return ret, nil
}
```
but it's really personal preference...

Reply:
If the source and the target are the same, then ret == src, no copy is done, and the original path is returned.

Reply:
true, but one of the err return cases explicitly returns `"", err` which contradicts the last comment. I guess that's my main point....